sites/cheerpx/src/content/docs/12-reference/13-CheerpX-Linux-setKmsCanvas.md

```diff
@@ -0,0 +1,61 @@
+---
+title: CheerpX.Linux#setKmsCanvas
+Description: Configure the KMS canvas for rendering graphical output in a CheerpX Linux instance.
+---
+
+```ts
+namespace CheerpX {
+	class Linux {
+		setKmsCanvas(
+			canvas: HTMLCanvasElement,
+			width: number,
+			height: number
+		): void;
+	}
+}
+```
+
+## Parameters
+
+- **canvas: HTMLCanvasElement** - The `canvas` element in your HTML document that serves as the rendering surface. This is where graphical output is displayed.
+
+- **width: number** - The width of the canvas in pixels.
+
+- **height: number** - The height of the canvas in pixels.
+
+## Description
+
+The `setKmsCanvas` method configures a kernel-mode setting (KMS) canvas, enabling the CheerpX Linux instance to render graphical. The dimensions of the canvas are defined by the `width` and `height` parameters, allowing the application to adapt to different display sizes dynamically.
+
+## Example
+
+```ts
+function setScreenSize(display) {
+	// Determine scaling factor
+	let scaleFactor = 1.0;
+	const displayWidth = display.offsetWidth;
+	const displayHeight = display.offsetHeight;
+
+	// Minimum width and height for the canvas
+	const minWidth = 1024;
```

Comment:
Let's simplify this code, the scaling factor logic was designed to make the alpine demo responsive, but it's not needed in general.